core/validation/validate.py

```diff
@@ -304,10 +308,13 @@ def validate_nullable(
 
     for column in sheet.columns:
         if column in non_nullable and (pd.isna(sheet[column]).any() | (sheet[column].astype("string") == "").any()):
+            failed_indexes = sheet.index[(pd.isna(sheet[column]) | (sheet[column].astype("string") == ""))]
+
             non_nullable_constraint_failure.append(
                 vf.NonNullableConstraintFailure(
                     sheet=sheet_name,
                     column=column,
+                    indexes=failed_indexes.asi8.tolist(),
```

Comment:
I think the .asi8 might be superfluous here, tolist() should handle casting to int without it?

Reply:
You're correct, .asi8 returns a np array, which I then had to cast to tolist() anyway